.github/workflows/jacktrip.yml

```diff
@@ -597,13 +580,105 @@ jobs:
           path: ${{ env.BUILD_PATH }}/${{ matrix.bundle-path }}
       - name: upload installer
         uses: actions/upload-artifact@v2
-        if: matrix.installer-path && (!startsWith(github.ref, 'refs/tags/') || (startsWith(github.ref, 'refs/tags/') && matrix.release-name)) && steps.check-secrets.outputs.available != 'true'
+        if: matrix.installer-path && (!startsWith(github.ref, 'refs/tags/') || (startsWith(github.ref, 'refs/tags/') && matrix.release-name)) && (steps.check-secrets.outputs.available != 'true' || runner.os == 'Windows')
```

Comment:
I'm getting lost in this conditions... but since we always want to upload the artifact now, wouldn't it be better to simply remove the check for secrets? 
```
if: matrix.installer-path && (!startsWith(github.ref, 'refs/tags/') || (startsWith(github.ref, 'refs/tags/') && matrix.release-name))
```

Reply:
No because one we named "signed" and the other we don't.

Reply:
ah, gotcha.